src/components/ha-selector/ha-selector-assist_pipeline.ts

```diff
@@ -0,0 +1,45 @@
+import { css, html, LitElement } from "lit";
+import { customElement, property } from "lit/decorators";
+import { AssistPipelineSelector } from "../../data/selector";
+import { HomeAssistant } from "../../types";
+import "../ha-pipeline-picker";
+
+@customElement("ha-selector-assist_pipeline")
```

Comment:
```suggestion
@customElement("ha-selector-assist-pipeline")
```
To be consistent with selector naming convention.
As a consequence the filename should also change.

Reply:
Selector used this naming convention. (e.g. `ha-selector-config_entry`, `ha-selector-color_temp`, `ha-selector-color_rgb`)

Reply:
That is true, I'm sorry for causing confusion.
I've only looked at the filenames which all use dashes instead of underscores.
Therefore only the filename should change.